README.md

```diff
@@ -238,3 +238,41 @@ pattern.
 [pytest]: https://docs.pytest.org/en/latest/
 [mozilla]: https://github.com/mozilla/addons-server
 [selenium-training]: https://qualitas-server.herokuapp.com/wiki/Introduction_to_Pytest_and_Selenium
+
+### HOW TO CREATE TEST DATA FOR REDIRECTS
+
+go to ce-scripts repo
+create virtual environment: 
+
+```pyenv virtualenv 3.7.x virtualenv_name```
+activate it: 
+
+```pyenv activate virtualenv_name```
+
+install requirements:
+
+```pip install -r requirements.txt```
+
+cd python
+
+run 
+
+```gen_book_uris.py <archive_host> <cnx_uuid>```
+
+e.g. $ ./gen_book_uris.py archive-staging.cnx.org e42bd376-624b-4c0f-972f-e0c57998e765
+
+this will create url variations as per rex_redirects.py and stores it in output/<cnx_uuid.txt> file 
+
+e.g. output/e42bd376-624b-4c0f-972f-e0c57998e765.txt
+
+rex_redirects.py file can be found here: /usr/local/lib/python3.7/site-packages/rex_redirects.py 
```

Comment:
I think using `/usr/local/lib/python3.7/site-packages/rex_redirects.py` explicitly like this is problematic since the exact path would be environment specific. Why not just embed a link to https://github.com/openstax/cnx-rex-redirects/blob/master/rex_redirects.py so readers can follow the reference that way?

Reply:
@rnathuji updated the readme file with the link.